src/lib/client/abstraction/org/projects.ts

```diff
@@ -0,0 +1,77 @@
+import { ClientError } from '../../errors/clientError';
+import {
+  ProjectsPostBodyType,
+  ProjectsPostResponseType,
+} from '../../generated/org';
+import { requestsManager } from 'snyk-request-manager';
+import { ProjectAttributes, ProjectsData } from './projectsTypes';
+
+const requestManager = new requestsManager({
+  burstSize: 100,
+  period: 100,
+  maxRetryCount: 10,
+  userAgentPrefix: 'snyk-api-ts-client',
+});
+
+export const getV3 = async (
+  classContext: Object,
+): Promise<ProjectsPostResponseType> => {
+  const response: ProjectsPostResponseType = {};
+  let url = '';
+  let urlQueryParams: Array<string> = ['version=2023-05-29', 'limit=10'];
```

Comment:
Is there potentially a reason to allow a customer to specify a version of the API they'd want to use?

Reply:
yes, but I think that would require v3 to be added as client formally, not in the abstraction here.